utils.js

```diff
@@ -4255,6 +4257,226 @@ export async function parseSetupPyFile(setupPyData) {
   return await parseReqFile(lines.join("\n"), false);
 }
 
+/**
+ * Method to create purl using information in pixi.lock file.
+ * According to pixi lock file satisfiability (https://pixi.sh/latest/features/lockfile/#lockfile-satisfiability)
+ *
+ *
+ *
+ * @param {*} packageData
+ * @returns
+ */
+function createPurlTemplate(packageData) {
+  const purlTemplate = `pkg:${packageData["kind"]}/${packageData["name"]}@${packageData["version"]}-${packageData["build"]}?os=${packageData["subdir"]}`;
+  return purlTemplate;
+}
+
+/**
+ * Method to parse pixi.lock data
+ *
+ * @param {Object} pixiData Contents of pixi.lock file
+ */
+export function parsePixiLockFile(pixiLockFileName, path) {
+  const pixiFileData = readFileSync(pixiLockFileName, { encoding: "utf-8" });
+  const pixiLockData = _load(pixiFileData);
+  // TODO: add atom slices, for allImports
```

Comment:
@aryan-rajoria I tend to discourage merging TODO comments into master. However, if these are things you're going to work on in the immediate future, they're acceptable. Ideally, I would like to see you open an issue for each TODO you leave and add the issue id as a reference in the TODO comment.

Reply:
Thank you for the feedback. I've addressed the TODOs by creating two corresponding GitHub issues and removing the comments.